minimacy: init at 0.6.2

jboynyc opened the pull request:
###### Description of changes

Adds a new package [minimacy](https://minimacy.net/), a compiler for an open-source minimalist computation system.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

wegank committed:
performous: 1.1 -> 1.2.0

JulienMalka reviewed (changes requested):
Hello, thanks for your contribution :) Could you add yourself as a maintainer for this package ? -> add yourself in the ``maintainers/maintainer-list.nix`` file and fill the ``meta.maintainer`` attribute.

JulienMalka commented on pkgs/development/compilers/minimacy/default.nix:
```diff
@@ -0,0 +1,60 @@
+  src = fetchFromGitHub {
+    owner = "ambermind";
+    repo = "minimacy";
+    rev = "refs/tags/${version}";
```
```suggestion
    rev =  version;
```

JulienMalka commented on pkgs/development/compilers/minimacy/default.nix:
```diff
@@ -0,0 +1,60 @@
+
+  src = fetchFromGitHub {
+    owner = "ambermind";
+    repo = "minimacy";
```
```suggestion
    repo = pname;
```

jboynyc committed:
minimacy: init at 0.6.2

jboynyc committed:
minimacy: init at 0.6.2

jboynyc committed:
maintainers: add jboy

jboynyc commented:
Thanks @JulienMalka for the guidance!

raphaelr commented on pkgs/development/compilers/minimacy/default.nix:
```diff
@@ -0,0 +1,62 @@
+
+  NIX_CFLAGS_COMPILE = "-Wno-unused-result";
+
+  buildPhase = ''
```
add `runHook preBuild` at the beginning, and `runHook postBuild` at the end.

mweinelt committed:
home-assistant: 2023.1.5 -> 2023.1.6

https://github.com/home-assistant/core/releases/tag/2023.1.6

wegank committed:
freedv: 1.8.6 -> 1.8.7

oxzi committed:
pythonPackages.pynmea2: 1.18.0 -> 1.19.0

https://github.com/Knio/pynmea2/compare/1.18.0...5d3d2013bff9c5bce2e14132d21fff865b1e58fd

Ma27 committed:
linux_6_0: drop

EOLed by upstream, doesn't receive any patches anymore, so let's drop
it.

Currently depends on #211886 which bumps the latest compatible ZFS
version to 6.1.

Also, clean up some old aliases.

misuzu committed:
fzf: fix failing test on 32-bit platforms

r-ryantm committed:
crowdin-cli: 3.9.1 -> 3.9.3

jboynyc committed:
maintainers: add jboy

jboynyc committed:
Merge remote-tracking branch 'refs/remotes/origin/master'

jboynyc committed:
Merge remote-tracking branch 'upstream/master'

flokli commented:
This mass-subscribed a bunch of people, who can't be unsubscribed. Please open a new PR, closing this one to prevent more noise.
